Resources/Prototypes/_NF/Reagents/chemicals.yml

```
@@ -4,3 +4,33 @@
   desc: reagent-desc-salicylicacid
   physicalDesc: reagent-physical-desc-powdery
   color: "#EEEEEE"
+
+- type: reagent
+  id: JuiceThatMakesYouOOOK
+  name: reagent-name-juicethatmakesyouoook
+  desc: reagent-desc-juicethatmakesyouoook
```

```suggestion
  name: reagent-name-ook
  desc: reagent-desc-ook
```
Take a page out of the juice that makes you weh and make this short and readable.